_posts/2020-06-11-trusted-web-activities.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: post
+title: Trusted Web Activities
+date: 2020-06-11
+tags: tutorial pwa PWA webapps spa android react
+author: Simon Bracegirdle
+image: /img/pwa.png
+---
+
+<center><img src="/img/pwa.png" /></center>
+<br/>
+
+# Trusted Web Activities
+
+A [Trusted Web Activity](https://developers.google.com/web/android/trusted-web-activity) (TWA) is a way to make your Progressive Web App (PWA) a first-class citizen on Android. It enables you to bundle your web app in an Android package and publish it to the Google Play Store.
+
+You don't need to make any drastic changes to your existing web app. You also don't need to install native development tools such as Android Studio, or write any native code.
+
+For example, let's say you have built an app that works well on mobile devices due to responsive web design. You can create an Android Package for that app via [PWA Builder](https://pwabuilder.com) and then publish that app to the Google Play Store.
+
+You have been able to integrate web apps with native apps for a while, but doing so has always been arduous. The previous approach was to create a wrapping native app and deal with the complexities of updating it separately to your web app.
+
+TWA's solve this problem by serving your app from the web with an actual browser, but without the address bar at the top or status bar at the bottom. This is like adding web app to your home screen from the in-browser feature, but with some differences that we will soon discover.
+
+### What is a Progressive Web App (PWA)?
```

Comment:
Look to link to our PWA capability report.

https://www.mechanicalrock.io/docs/case-studies/pwa-capability-report.pdf